src/middlewared/middlewared/etc_files/local/users.oath.mako

```diff
@@ -4,9 +4,7 @@
 	twofactor_auth = render_ctx['auth.twofactor.config']
 	users = middleware.call_sync(
```

Comment:
Since you're here already, I think we should change these yoda style lines since I don't find them intuitive a bit hard to read. i.e.
```suggestion
	users = []
	if twofactor_auth['enabled']:
	    users = middleware.call_sync('auth.twofactor.get_users_config')
```